docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -0,0 +1,221 @@
+----
++
+[source,bash]
+----
+NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+hot-restart-persistence-hazelcast-0   Bound    pvc-116b4084-a436-4462-b413-511b77df307b   20Gi       RWO            standard       6m
+hot-restart-persistence-hazelcast-1   Bound    pvc-a7711b6b-dcbf-45cb-9577-8ce6b1892f2f   20Gi       RWO            standard       5m
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       5m
+
+----
+
+. Reapply the `Hazelcast` resource:
++
+[source,bash]
+----
+kubectl apply -f ./hazelcast-persistence.yaml
+----
+
+. See that the same PVCs are bound to the new pods:
++
+[source,bash]
+----
+kubectl get pods -l app.kubernetes.io/instance=hazelcast -ojsonpath='{.items[*].spec.volumes[?(@.name=="hot-restart-persistence")].persistentVolumeClaim.claimName}'
+
+----
++
+You should see something like the following:
++
+[source,bash]
+----
+hot-restart-persistence-hazelcast-0 hot-restart-persistence-hazelcast-1 hot-restart-persistence-hazelcast-2
+----
+
+[[recovery-policy]]
+== Choosing a Cluster Recovery Policy
+
+To decide how a cluster should behave when one or more members cannot rejoin after a cluster-wide restart, you can define one of the following cluster recovery policies. The Operator supports all the policies in the Hazelcast Platform `cluster-data-recovery-policy configuration` options. For complete descriptions and advice on choosing a policy, see the xref:hazelcast:storage:configuring-persistence.adoc#policy[Platform documentation].
+
+[cols="1,1"]
+|===
+| ```FullRecoveryOnly```
+| Does not allow partial start of the cluster.
+
+| ```PartialRecoveryMostRecent```
+| Allows partial start with the members that have most recent partition table.
+
+| ```PartialRecoveryMostComplete```
+| Allows partial start with the member that have most complete partition table.
+|===
+
+=== Configuring Force-Start
+
+If you use the `FullRecoveryOnly` policy, you can configure the Operator to detect failed Hazelcast members and automatically trigger a force-start. The Operator will trigger a force-start only if the cluster is in a `PASSIVE` state.
+
+WARNING: The cluster loses all persisted data after a force-start.
+
+Enable `autoForceStart`:
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast-autoforcestart.yaml[]
+----
+
+[[hostpath]]
+== HostPath Support for Persistence
+
+You can also use HostPath to enable persistence.
+
+WARNING: HostPath support is discouraged for the production environments for the reasons
+mentioned in the link:https://kubernetes.io/docs/concepts/storage/volumes/#hostpath[Kubernetes documentation]
+
+WARNING: HostPath support expects the size of the cluster to be equal to the number of Kubernetes nodes
+and pods are distributed to the nodes equally. You can manage it by setting `topologySpreadContraints` field.
```

Comment:
```suggestion
WARNING: HostPath support expects the size of the cluster to be equal to the number of Kubernetes nodes
and pods are distributed to the nodes equally. You can manage it by setting `topologySpreadContraints` field described xref:scheduling-configuration.adoc[here].
```

I am not sure if my usage of `xref` is correct. I also wanted to give `#topology-spread-constraints` but not sure how to do it. @JakeSCahill, do you know how?

Reply:
That looks right, you can add anchors to the end like so: `xref:scheduling-configuration.adoc#topology-spread-constraints[]`.

I'd suggest using descriptive text for the link - something like 'You can manage how pods are distributed among nodes by setting the `topologySpreadContraints` field, which is described in xref:scheduling-configuration.adoc#topology-spread-constraints[Scheduling Hazelcast Pods]'.

Reply:
Sure, I will change it to that. Thanks :smile: